src/javascript/crypto/e2e/vrf/sha3shake256.js

```diff
@@ -0,0 +1,353 @@
+/**
+ * @license
+ * Copyright 2015 Yahoo Inc. All rights reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+/**
+ * @fileoverview Extends js-sha3 to support SHAKE256, and outputs byte array
+ */
+
+goog.provide('e2e.vrf.sha3Shake256');
+
+goog.require('goog.asserts');
+
+
```

Comment:
all the variable declarations in the top-level namespace below should be in the e2e.vrf.sha3Shake256 namespace